src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesBackupContainer.cs

```diff
@@ -41,7 +41,7 @@ public class GetAzureRmRecoveryServicesBackupContainer : RSBackupVaultCmdletBase
         [Parameter(Mandatory = false, Position = 2,
             HelpMessage = ParamHelpMsgs.Container.BackupManagementType)]
         [ValidateNotNullOrEmpty]
-        [ValidateSet("AzureVM", "MARS", "AzureSQL", "AzureStorage")]
+        [ValidateSet("AzureVM", "MARS", "AzureWorkload", "AzureStorage")]
```

Comment:
this looks like breaking change to me, you can keep AzureSql and add AzureWorkload